test/unit/region.spec.js

```diff
@@ -466,6 +497,65 @@ describe('region', function() {
         });
       });
     });
+
+    describe('when passing the "replaceElement" option', function() {
+      beforeEach(function() {
+        // empty region to clean existing view
+        this.myRegion.empty();
+
+        this.$parentEl = this.myRegion.$el.parent();
+        this.regionHtml = this.$parentEl.html();
+        this.showOptions = {replaceElement: true};
+        this.myRegion.show(this.view, this.showOptions);
+      });
+
+      it('should append the view HTML to the parent "el"', function() {
+        expect(this.$parentEl).to.contain.$html(this.view.$el.html());
+      });
+
+      it('should remove the region\'s "el" from the DOM', function() {
+        expect(this.$parentEl).to.not.contain.$html(this.regionHtml);
+      });
+
+      describe('and then emptying the region', function() {
+        beforeEach(function() {
+          this.myRegion.empty();
+        });
+
+        it('should remove the view from the parent', function() {
+          expect(this.$parentEl).to.not.contain.$html(this.view.$el.html());
+        });
+
+        it('should restore the region\'s "el" to the DOM', function() {
+          expect(this.$parentEl).to.contain.$html('<div id="region"></div>');
+        });
+      });
+
+      describe('and showing another view', function() {
+        beforeEach(function() {
+          this.MyView2 = Marionette.View.extend({
```

Comment:
This is `next` so `View === ItemView === LayoutView`?  and the old `View` is now `AbstractView`